orchagent/neighorch.cpp

```diff
@@ -30,6 +30,14 @@ bool NeighOrch::addNextHop(IpAddress ipAddress, string alias)
 {
     SWSS_LOG_ENTER();
 
+    Port p;
+    if (!gPortsOrch->getPort(alias, p))
```

Comment:
The port doesn't exist because...?

Reply:
Since PortsOrch::getPort() has return value, it is a safety check.

Reply:
I would recommend to assert then.